Source/Dafny/Translator.cs

```diff
@@ -1067,55 +1064,29 @@ internal class PredefinedDecls {
     }
 
     /// <summary>
-    /// adding TraitParent axioms
-    /// if a class A extends trait J and B does not extend anything, then this method adds the followings to the sink:
-    ///   axiom TraitParent(class.A) == class.J;
-    ///   axiom TraitParent(class.B) == NoTraitAtAll;
+    /// For every revealed type (class or trait) C that extends a trait J, add:
```

Comment:
I know the AST supports traits extending traits, but AFAICT the parser doesn't. I think it's fine to be forwards-compatible here, but probably worth mentioning it in the comment to avoid confusion.